stories/icon-tab-bar/icon-tab-bar.stories.js

```diff
@@ -0,0 +1,1434 @@
+export default {
+    title: 'Components/Icon Tab Bar',
+    parameters: {
+        description: `
+The Icon Tab Bar can be used for navigation within an object, or as a filter. Each tab of the component links to a different content area or view.
+
+##Usage
+**Use the icon tab bar if:**
+
+- Your business objects need to show multiple facets at the same time.
+- You want to allow the user to browse through these facets.
+- You need a prominent or very visual filter on top of a list.
+- You have clear-cut process steps that need to be visualized.
+
+**Do not use the icon tab bar if:**
+
+- You plan to use only one single tab.
+
+##Semantic colors
+Only use semantic colors if it is important for users to know that they need to take action (for example, to indicate errors or critical situations requiring action). Otherwise, use the neutral default colors. To add semantic color to a tab use a modifier class from the table below on <code>fd-icon-tab-bar\\_\\_item</code> level.
+
+| **Color** | **Modifier class** |
+| :--------- | :---------- |
+| positive &nbsp;&nbsp; | <code>fd-icon-tab-bar\\_\\_item--positive</code> |
+| negative &nbsp;&nbsp; | <code>fd-icon-tab-bar\\_\\_item--negative</code> |
+| critical &nbsp;&nbsp; | <code>fd-icon-tab-bar\\_\\_item--critical</code> |
+| informative &nbsp;&nbsp; | <code>fd-icon-tab-bar\\_\\_item--informative</code> |
+
+##Horizontal paddings
+You can add horizontal paddings by applying a modifier class to the container. For responsive horizontal paddings (based on the screen size) add the <code>fd-icon-tab-bar--responsive-paddings</code> modifier class. In this case the left and rigth spacing will change as the screen size changes.
+
+| **Size** | **Modifier class** |
+| :--------- | :---------- |
+| sm (1rem) &nbsp;&nbsp; | <code>fd-icon-tab-bar--sm</code> |
+| md (2rem) &nbsp;&nbsp; | <code>fd-icon-tab-bar--md</code> |
+| lg (2rem) &nbsp;&nbsp; | <code>fd-icon-tab-bar--lg</code> |
+| xl (3rem) &nbsp;&nbsp; | <code>fd-icon-tab-bar--xl</code> |
+| xxl (3rem) &nbsp;&nbsp; | <code>fd-icon-tab-bar--xxl</code> |
+
+##Background
+By default the background for the Icon Tab Bar header and content is set to "solid". This will apply <code>--sapObjectHeader_Background</code> background color and <code>--sapContent_HeaderShadow</code> box-shadow to the header and <code>--sapBackgroundColor</code> to the container (panel). It can be changed to "translucent" or "transparent" using modifier classes. In translucent mode the header gets <code>--sapObjectHeader_Background</code> background color and the panel <code>--sapGroup_ContentBackground</code> background color. 
```

Comment:
Rewrite: By default, the background for the Icon Tab Bar header and content is set as "solid".